### PR TITLE
Add method to get completed list and save to localStorage

### PR DESCRIPTION
**Pull request by mustabbas**
### Change Done
- [ ] Added a new JavaScript file and import it as a module it will contain methods related to the status updates (completed: true/false).
- [ ] Added event listener to the checkbox (change).
- [ ] Updated items object's value for completed key upon user actions.
- [ ] Stored the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -30,6 +34,19 @@ function AddListItem() {
   }
 }
 
+function checkBox() {
+  const checkboxAll = document.querySelectorAll('.checkBox');
+  for (let i = 0; i < checkboxAll.length; i += 1) {
+    // eslint-disable-next-line no-loop-func
+    checkboxAll[i].addEventListener('click', () => { completeTask(checkboxAll[i].value, ListArray); });
+  }
+}
```
Comment (henatan99) [inline src/index.js]: - [ ] When a checked box is unchecked it doesn't update the `isCompleted` value. Kindly make it work. :pray: 

Screenshot from 2021-09-24 02-11-23

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @mustabbas , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] OPTIONAL: I recommend you commit your changes more frequently as it helps track your changes better. Kindly apply this to your future commit. :pray: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (mustabbas) [reply src/index.js]: Hello thanks for your feedback I am not getting this problem working very good with me  here is the live demo 
https://mustabbas.github.io/TodoList/dist/
Screen Shot 2021-09-23 at 11 37 16 PM

Comment (henatan99) [reply src/index.js]: Hi @mustabbas , can you please try to uncheck the already checked box and see if it changes the `isCompleted` value to false. :pray:

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @mustabbas, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

200w


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**